.env-example

```diff
@@ -1,2 +1,4 @@
 PORT=
-ATLAS_URI=
\ No newline at end of file
+ATLAS_URI=
+COINMARKETCAP_API_KEY=
+GRINDERY_MERCARI_CLIENT=
```

Comment:
We need to add the `GRINDERY_MERCARI_CLIENT` env to pass the ci/cd

Reply:
@juniormp Yes sure, send me via DM on Slack :) will add to GCP